<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleWindow</class>
 <widget class="QMainWindow" name="ScheduleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>302</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>612</width>
    <height>302</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>612</width>
    <height>302</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="nnameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="enabledCheckBox">
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="beginLabel">
          <property name="text">
           <string>Begin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="beginTimeEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="endLabel">
          <property name="text">
           <string>End</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="endTimeEdit"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="mondayCheckBox">
          <property name="text">
           <string>Mo.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tuesdayCheckBox">
          <property name="text">
           <string>Tu.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="wednesdayCheckBox">
          <property name="text">
           <string>We.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="thursdayCheckBox">
          <property name="text">
           <string>Th.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fridayCheckBox">
          <property name="text">
           <string>Fr.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="saturdayCheckBox">
          <property name="text">
           <string>Sa.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sundayCheckBox">
          <property name="text">
           <string>Su.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="repeatCheckBox">
        <property name="text">
         <string>Repeat</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="typeComboBox">
          <item>
           <property name="text">
            <string>Free</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Downloading</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ask not to download</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="onConflictBehaviourLabel">
          <property name="text">
           <string>On conflict behaviour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="onConflictBehaviourComboBox">
          <item>
           <property name="text">
            <string>Ignore</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Notify</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Abandon</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="removeOnEndCheckBox">
        <property name="text">
         <string>Remove on end</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>Confirm</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
